odolbeau/phone-number-bundle/3.0/config/packages/misd_phone_number.yaml

```diff
@@ -0,0 +1,6 @@
+# To persist libphonenumber\PhoneNumber objects, add the Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType mapping to your application's config.
+# This requires: doctrine/doctrine-bundle
+doctrine:
+    dbal:
+        types:
+            phone_number: Misd\PhoneNumberBundle\Doctrine\DBAL\Types\PhoneNumberType
```

Comment:
this should be commented imo, as doctrine is not always installed and is optionnal.